server/src/main/java/io/crate/analyze/TableElementsAnalyzer.java

```diff
@@ -532,6 +533,8 @@ public Void visitColumnDefinition(ColumnDefinition<?> node, ColumnIdent parent)
                 }
                 Symbol defaultSymbol = expressionAnalyzer.convert(defaultExpression, expressionContext);
                 builder.defaultExpression = defaultSymbol.cast(builder.type, CastMode.IMPLICIT);
+                // only used to validate; result is not used to preserve functions like `current_timestamp`
+                normalizer.normalize(builder.defaultExpression, txnCtx);
```

Comment:
What's the purpose of those? I tried disabling them and there is not test failure.

Reply:
`test_alter_table_cannot_add_broken_generated_column` fails without this because without the `_cast` the `DocTableInfoFactory` analysis behaves slightly different

Reply:
Thank you but weird, I've run a clean test on everything and didn't have a failure, but now I run from intellij and verified, apologies.